NuGet.config

```diff
@@ -9,6 +9,7 @@
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <add key="darc-pub-dotnet-core-setup-916b5cb" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-916b5cba/nuget/v3/index.json" />
     <add key="darc-pub-dotnet-aspnetcore-c3acdca-2" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-aspnetcore-c3acdcac-2/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-aspnetcore-b67a903" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-aspnetcore-b67a9034/nuget/v3/index.json" />
```

Comment:
revert this

Reply:
if you cannot build without this change. Something wrong with existing 3.1.4xx darc setup. And we need to fix it going forward

Reply:
Yup, will do. I cherry-picked the change from 3.1.300, but this should be on nuget.org now